code/modules/vehicles/mecha/equipment/mecha_equipment.dm

```diff
@@ -54,10 +60,11 @@
 	. = ..()
```

Comment:
immediatly in the next line check this return value and return if true i.e.
```
. = ..()
if(.)
   return
```

We want this client side code to be executed only if all the conditions are met. for example https://github.com/tgstation/tgstation/blob/382b8c12c1d0cd9cae2bd3014f72efba0c1928f8/code/game/objects/items/rcd/RCD.dm#L461-L464. 

One time i forgot to do the early return and ghosts could suddenly mess with the rcd. Just an example

Reply:
This sounds like a feature 😄